# Marathi translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1076
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-06-24 15:42-0700\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Brian Murray <brian@ubuntu.com>\n"
"Language-Team: Marathi <mr@li.org>\n"
"Language: mr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:47+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "%s साठी सर्व्हर"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "मुख्य सर्व्हर"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "कस्टम सर्व्हर्स"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "sources.list नोंदींची गणना करू शकत नाही"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"पॅकेज फाईल्स मिळाल्या नाहीत, बहुतेक हि उबुंटूची CD नाही किंवा architecture वेगळे आहे."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "CD घेऊ शकले नाही."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD घेताना चूक झाली आहे, अपग्रेड थांबवला जात आहे. जर ही उबुंटूची CD आहे तर हा बग म्हणून "
"आम्हाला कळवा. \n"
"\n"
"चुकीबाबतचा संदेश: \n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:180
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "खराब स्थितीत असलेले पॅकेज काढून टाका"
msgstr[1] "खराब स्थितीत असलेले पॅकेजेस काढून टाका"

#: ../DistUpgrade/DistUpgradeCache.py:183
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"पॅकेज '%s' हे अव्यवस्थित स्थितीमध्ये आहे आणि ते परत प्रस्थापित करावे लागेल, पण त्याच्यासाठीचे "
"archive मिळत नाही आहे. पुढेजाण्यासाठी हे पॅकेज काढून टाकावे का?"
msgstr[1] ""
"पॅकेजेस '%s' ही अव्यवस्थित स्थितीमध्ये आहेत आणि ती परत प्रस्थापित करावी लागतील, पण "
"त्यासाठीचे archive मिळत नाही आहे. पुढेजाण्यासाठी ही पॅकेजस काढून टाकावीत का?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:299
msgid "The server may be overloaded"
msgstr "कदाचित सर्व्हर वर अतिरिक्त भार असावा"

#: ../DistUpgrade/DistUpgradeCache.py:412
msgid "Broken packages"
msgstr "अपुर्ण पॅकेजेस"

#: ../DistUpgrade/DistUpgradeCache.py:413
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"तुमच्या सिस्टिमवर बरीच अपुर्ण/तुटलेली पॅकेजेस आहेत, जी या सॉफ्टवेअर सह दुरुस्त होवू शकणार "
"नाहीत. पुढे चालू ठेवण्याअगोदर प्रथमतः synaptic किंवा apt-get द्वारे ती अपुर्ण/तुटलेली "
"पॅकेजेस दुरुस्त करा."

#: ../DistUpgrade/DistUpgradeCache.py:649
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "अपुर्ण पॅकेजेस"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:658
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:676
msgid "This is most likely a transient problem, please try again later."
msgstr "ही तात्पुरती समस्या आहे, काही क्षणांनंतर पुन्हा प्रयत्न करा."

#: ../DistUpgrade/DistUpgradeCache.py:679
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:681
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "Could not calculate the upgrade"
msgstr "अपग्रेडची गणना करू शकले नाही"

#: ../DistUpgrade/DistUpgradeCache.py:746
msgid "Error authenticating some packages"
msgstr "काही पॅकेजेसचे अधिप्रमाणन करताना त्रुटी येत आहेत"

#: ../DistUpgrade/DistUpgradeCache.py:747
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"काही पॅकेजेसचे अधिप्रमाणन करणे शक्य नव्हते. बहुदा हे तात्पुरत्या नेटवर्क समस्येमुळे झाले असावे. "
"काही वेळाने पुन्हा प्रयत्न करून पहा. काही अधिप्रमाणित न केलेल्या पॅकेजेसची यादी पाहण्यासाठी "
"खाली पहा."

#: ../DistUpgrade/DistUpgradeCache.py:768
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"'%s' हे पॅकेज काढून टाकण्यासाठी चिन्हांकित केले आहे पण ते काढून टाकलेल्या ब्लॅकलिस्ट मध्ये आहे."

#: ../DistUpgrade/DistUpgradeCache.py:774
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "'%s' हे अत्यावश्यक पॅकेज काढून टाकण्यासाठी चिन्हांकित केलेले आहे."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "ब्लॅकलिस्ट केलेली '%s' आवृत्ती स्थापित करण्याचा प्रयत्न करीत आहे"

#: ../DistUpgrade/DistUpgradeCache.py:905
#, python-format
msgid "Can't install '%s'"
msgstr "'%s' हे स्थापित करू शकत नाही"

#: ../DistUpgrade/DistUpgradeCache.py:906
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"एक आवश्यक संकुल प्रतिष्ठापीत करणे अशक्य होते. टर्मिनल मध्ये 'ubuntu-bug ubuntu-release-"
"upgrader-core' टाकून बगची तक्रार करा."

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid "Can't guess meta-package"
msgstr "meta-package ओळखू शकत नाही"

#: ../DistUpgrade/DistUpgradeCache.py:918
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "कॅशे (cache) वाचत आहे"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid "Unable to get exclusive lock"
msgstr "एक्स्क्ल्युजिव्ह लॉक मिळविण्यात असमर्थ"

#: ../DistUpgrade/DistUpgradeController.py:227
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"कदाचित तुमचे दुसरे एखादे पॅकेज मॅनेजमेंट ऍप्लिकेशन (उदा. apt-get किंवा aptitude) चालू आहे. "
"कृपया ते ऍप्लिकेशन प्रथम बंद करा. (कारण apt-get किंवा aptitude द्वारे एकाच वेळी "
"एकापेक्षा अधिक पॅकेजेस स्थापित करता येत नाहीत.)"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid "Upgrading over remote connection not supported"
msgstr "रीमोट कनेक्शन वरून अपग्रेड करण्यास समर्थन नाही"

#: ../DistUpgrade/DistUpgradeController.py:286
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"तुम्ही रीमोट ssh कनेक्शन वरून एका फ्रन्टएन्ड सहाय्यकासह अपग्रेड चालवत आहात ज्याचे तो "
"फ्रन्टएन्ड सहाय्यक समर्थन करीत नाही. कृपया टेक्स्ट मोड मधून 'do-release-upgrade' या "
"कमांडद्वारे अपग्रेड करून पहा.\n"
"\n"
"अपग्रेड आता थांबविले जाईल. कृपया ssh च्या शिवाय प्रयत्न करून पहा."

#: ../DistUpgrade/DistUpgradeController.py:300
msgid "Continue running under SSH?"
msgstr "SSH च्या अंतर्गत चालू ठेवायचे का?"

#: ../DistUpgrade/DistUpgradeController.py:301
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"असे दिसतेय की हे सत्र ssh च्या अंतर्गत चालू आहे. सध्यातरी ssh वरून अपग्रेड चालवणे धोकादायक "
"असू शकते कारण जर ही क्रिया अयशस्वी झाली तर पुनःस्थितीत येणे अतिशय अवघड असेल.\n"
"\n"
"जर तुम्ही चालू ठेवले, तर '%s' या पोर्टवर एक शिल्ल्कचे ssh daemon सुरू होईल.\n"
"तुम्ही पुढे चालू ठेवू इच्छिता का?"

#: ../DistUpgrade/DistUpgradeController.py:315
msgid "Starting additional sshd"
msgstr "शिल्लकचे sshd सुरू करीत आहे"

#: ../DistUpgrade/DistUpgradeController.py:316
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"जर काही कारणास्तव ही क्रिया अयशस्वी झाली तर सुलभ रिकव्हरी करण्यासाठी '%s' या "
"पोर्टवर एक शिल्लकचे sshd चालू होईल. सध्या चालू असलेल्या ssh सोबत जरी काही अनुचित "
"प्रकार घडलाच, तरीही तुम्ही एका शिल्लकच्या sshd ला कनेक्ट करू शकता.\n"

#: ../DistUpgrade/DistUpgradeController.py:324
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"तुम्ही फायरवॉल वापरत असाल तर तुम्हाला हे पोर्ट तात्पुरते उघडावे लागेल. हे धोकादायक ठरु "
"शकत असल्याने आपोआप केले जात नाही. पोर्ट उघडण्यासाठी तुम्ही उदा.\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:402
#: ../DistUpgrade/DistUpgradeController.py:426
msgid "Can not upgrade"
msgstr "अपग्रेड करू शकत नाही"

#: ../DistUpgrade/DistUpgradeController.py:403
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "या साधनासह '%s' पासून '%s' कडे अपग्रेड करण्यास समर्थन नाही."

#: ../DistUpgrade/DistUpgradeController.py:427
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid "Can not write to '%s'"
msgstr "'%s'  वर लिहु शकत नाही"

#: ../DistUpgrade/DistUpgradeController.py:457
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"तुमच्या कॉम्प्युटरवरच्या सिस्टिम डिरेक्टरी '%s' वर लिहिणे शक्य नाहीये. अपग्रेड पुढे जावू शकत "
"नाही.\n"
"सिस्टिम डिरेक्टरीवर लिहिणे शक्य आहे याची कृपया खातरजमा करावी."

#: ../DistUpgrade/DistUpgradeController.py:468
msgid "Include latest updates from the Internet?"
msgstr "इंटरनेटवरून सध्या उपलब्ध असलेले ताजे अद्ययावत जोडायचे का?"

#: ../DistUpgrade/DistUpgradeController.py:469
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"जर तुमच्याकडे नेटवर्क कनेक्शन असेल तर अपग्रेड चालू असताना अपग्रेड प्रणाली इंटरनेटवरून उपलब्ध "
"ताजे अद्ययावत स्वयंचलितपणे आपोआप डाउनलोड करेल.\n"
"\n"
"अपग्रेड करण्यास बराच कालावधी लागू शकतो, पण जेव्हा अपग्रेड पूर्ण होईल तेव्हा तुमची सिस्टिम "
"आजपर्यंत पूर्णपणे अद्ययावत स्थितीत असेल.  तुम्ही हे निवडणे टाळू देखील शकता, पण अपग्रेड "
"झाल्यानंतर तुम्हाला ताजे अद्ययावत स्थापित करावेच लागतील.\n"
"जर तुम्ही येथे 'नाही' असे उत्तर दिले, तर अपग्रेड प्रणाली तुमच्या नेटवर्कचा मुळीच उपयोग "
"करणार नाही."

#: ../DistUpgrade/DistUpgradeController.py:647
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "प्रकाशन %s विकास टप्प्यादरम्यान माणसांसाठी नाही"

#: ../DistUpgrade/DistUpgradeController.py:748
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%s साठी अपग्रेड अकार्यान्वीत केलेले आहे"

#: ../DistUpgrade/DistUpgradeController.py:805
msgid "No valid sources.list entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:806
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:818
msgid "No valid mirror found"
msgstr "योग्य प्रतिमा सापडली नाही"

#: ../DistUpgrade/DistUpgradeController.py:819
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:839
msgid "Generate default sources?"
msgstr "डिफॉल्ट सोर्सेस बनवायचे?"

#: ../DistUpgrade/DistUpgradeController.py:840
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"तुमच्या 'sources.list'च्या छाननीत '%s' ची एकही वैध नोंद मिळाली नाही.\n"
"\n"
"'%s'च्या डिफॉल्ट नोंदी लिहायच्या का? तुम्ही जर 'No' म्हणालात तर अपग्रेड रद्द होईल."

#: ../DistUpgrade/DistUpgradeController.py:875
#: ../DistUpgrade/DistUpgradeController.py:881
msgid "Repository information invalid"
msgstr "Repository/भांडारातली माहिती अवैध"

#: ../DistUpgrade/DistUpgradeController.py:876
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Repository/भांडारातली माहिती अपग्रेड करताना फाईल अवैध बनली म्हणुन बग रिपोर्ट "
"करण्याची क्रिया सुरु होत आहे"

#: ../DistUpgrade/DistUpgradeController.py:882
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:891
msgid "Third party sources disabled"
msgstr "तृतीय पक्षीय/Third party स्त्रोत निष्क्रिय"

#: ../DistUpgrade/DistUpgradeController.py:892
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"तुमच्या sources.list मधील काही तृतीय पक्षीय/Third party स्त्रोत निष्क्रिय केलेले आहेत. "
"अपग्रेड झाल्यानंतर् तुम्ही 'software-properties' टूल किंवा पॅकेज मॅनेजरने ते पुन: सक्रीय करु "
"शकता."

#: ../DistUpgrade/DistUpgradeController.py:934
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "पैकेज अपुर्णावस्थेत आहे."
msgstr[1] "पैकेजेस अपुर्णावस्थेत आहेत."

#: ../DistUpgrade/DistUpgradeController.py:937
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] "पैकेज अपुर्णावस्थेत आहे आणि पुन:प्रस्थापित करावे लागेल,"
msgstr[1] "पैकेजेस अपुर्णावस्थेत आहेत आणि पुन:प्रस्थापित करावी लागतील,"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid "Error during update"
msgstr "नवीकरण करताना त्रुटि."

#: ../DistUpgrade/DistUpgradeController.py:1001
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"अपडेट करताना अडचण निर्माण झाली. हा सामान्यत: नेटवर्क मधील दोष असतो, तुमचे नेटवर्क "
"कनेक्शन तपासा आणि परत प्रयत्न करा."

#: ../DistUpgrade/DistUpgradeController.py:1044
#: ../DistUpgrade/DistUpgradeController.py:1049
msgid "EFI System Partition (ESP) not usable"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1045
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1050
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1057
msgid "Not enough free disk space"
msgstr "डिस्कवर पुरेशी जागा उपलब्ध नाही"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1060
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1065
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1067
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1072
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1075
msgid "Reboot to clean up files in /tmp."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1114
#: ../DistUpgrade/DistUpgradeController.py:1990
#: ../DistUpgrade/DistUpgradeController.py:2040
msgid "Calculating the changes"
msgstr "फ़रकाची मोजदाद करत आहे"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1158
msgid "Do you want to start the upgrade?"
msgstr "अपग्रेड सुरु करायचा?"

#: ../DistUpgrade/DistUpgradeController.py:1177
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid "Upgrade canceled"
msgstr "अपग्रेड रद्द झाला"

#: ../DistUpgrade/DistUpgradeController.py:1250
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"आता अपग्रेड रद्द होईल आणि तुमची सिस्टिम होती तशी पुर्ववत होईल. तुम्ही नंतर परत कधीही "
"अपग्रेड सुरु करु शकता."

#: ../DistUpgrade/DistUpgradeController.py:1256
#: ../DistUpgrade/DistUpgradeController.py:1401
msgid "Could not download the upgrades"
msgstr "अपग्रेड डाउनलोड होउ शकला नाही"

#: ../DistUpgrade/DistUpgradeController.py:1257
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"अपग्रेडच्या बाहेर पडलो आहे. कृपया तुमचे इंटरनेट कनेक्शन अगर इनस्टॉलेशन माध्यम तपासा आणि परत "
"प्रयत्न करा. आत्तापर्यंत डाउनलोड केलेल्या सर्व फाईल्स तशाच ठेवलेल्या आहेत."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1350
#: ../DistUpgrade/DistUpgradeController.py:1389
#: ../DistUpgrade/DistUpgradeController.py:1505
msgid "Error during commit"
msgstr "कमिट करताना अडचण"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1391
#: ../DistUpgrade/DistUpgradeController.py:1544
msgid "Restoring original system state"
msgstr "सिस्टिम मुळच्या स्थितीला परत आणत आहे"

#: ../DistUpgrade/DistUpgradeController.py:1353
#: ../DistUpgrade/DistUpgradeController.py:1368
#: ../DistUpgrade/DistUpgradeController.py:1392
msgid "Could not install the upgrades"
msgstr "अपग्रेड स्थापन करु शकलो नाही"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1358
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"अपग्रेडच्या बाहेर पडलो आहे. तुमची सिस्टिम निरुपयोगी स्थितीत असु शकते. पुर्वपदावर येण्याचा "
"प्रयत्न केला जाईल (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1363
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1402
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"अपग्रेडच्या बाहेर पडलो आहे. कृपया तुमचे इंटरनेट कनेक्शन अगर इनस्टॉलेशन माध्यम तपासा आणि परत "
"प्रयत्न करा. "

#: ../DistUpgrade/DistUpgradeController.py:1430
msgid "Searching for obsolete software"
msgstr "कालबह्या सॉफ्टवेअरसाठी शोध करत आहे"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "Remove obsolete packages?"
msgstr "कालबाह्य पॅकेजेस काढायची?"

#: ../DistUpgrade/DistUpgradeController.py:1494
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "जपवून ठेवा (_K)"

#: ../DistUpgrade/DistUpgradeController.py:1494
msgid "_Remove"
msgstr "काढून टाका (_R)"

#: ../DistUpgrade/DistUpgradeController.py:1506
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "साफ़-सफ़ाई करताना अडचण आली. अधिक माहितीसाठी कृपया खालील संदेश पाहा. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1582
msgid "Required depends is not installed"
msgstr "आवश्यक निर्भर पॅकेजेस इनस्टॉल नाहियेत"

#: ../DistUpgrade/DistUpgradeController.py:1583
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "आवश्यक निर्भर पॅकेज '%s' इनस्टॉल नाहिये. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1850
#: ../DistUpgrade/DistUpgradeController.py:1940
msgid "Checking package manager"
msgstr "पॅकेज मॅनेजर तपासत आहे"

#: ../DistUpgrade/DistUpgradeController.py:1856
#: ../DistUpgrade/DistUpgradeController.py:1862
msgid "Preparing the upgrade failed"
msgstr "अपग्रेडची तयारी अयशस्वी"

#: ../DistUpgrade/DistUpgradeController.py:1857
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"सिस्टिमला अपग्रेडसाठी तयार करण्यात अपयश आले म्हणुन बग रिपोर्टींगची क्रिया सुरु करत आहे."

#: ../DistUpgrade/DistUpgradeController.py:1863
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1880
#: ../DistUpgrade/DistUpgradeController.py:1890
msgid "Getting upgrade prerequisites failed"
msgstr "अपग्रेडसाठीचे पुर्वावश्यक मिळवणे अयशस्वी"

#: ../DistUpgrade/DistUpgradeController.py:1881
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"सिस्टिम अपग्रेडसाठीचे पुर्वावश्यक मिळवण्यात अयशस्वी झाली. अपग्रेड आता बाहेर पडेल आणि तुमची "
"सिस्टिम मूळ स्थितीला आणुन ठेवेल.\n"
"\n"
"याशिवाय एक बग रिपोर्टिंग क्रियादेखील सुरु करण्यात येत आहे."

#: ../DistUpgrade/DistUpgradeController.py:1891
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1924
msgid "Updating repository information"
msgstr "repository/कोषाची माहिती अपडेट करत आहे"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Failed to add the cdrom"
msgstr "cdrom समाविष्ट करण्यात अपयश"

#: ../DistUpgrade/DistUpgradeController.py:1932
msgid "Sorry, adding the cdrom was not successful."
msgstr "क्षमस्व, cdrom चा समावेश करण्यात यश आले नाही."

#: ../DistUpgrade/DistUpgradeController.py:1967
msgid "Invalid package information"
msgstr "पॅकेजची माहिती अवैध"

#: ../DistUpgrade/DistUpgradeController.py:1968
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"तुमचा संकुल माहिती अद्यतनित केल्यानंतर, अत्यावश्यक पॅकेज '%s' स्थित केले जाऊ शकत नाही. याचे "
"कारण आपल्याकडे सॉफ्टवेअर स्रोतात सूचीबद्ध अधिकृत मिरर नाही आहे, किंवा कारण आपण वापरत "
"असलेल्या मिररवर जास्त भार आहे. संरचीत सॉफ्टवेअर स्त्रोतांची सध्याची यादी /etc/apt/"
"sources.list मध्ये पहा.\n"
"एक ओव्हरलोडेड मिरर बाबतीत, आपण नंतर अद्यतनाचा पुन्हा प्रयत्न करू शकता."

#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2096
msgid "Fetching"
msgstr "आणतोय"

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2025
#: ../DistUpgrade/DistUpgradeController.py:2046
#: ../DistUpgrade/DistUpgradeController.py:2100
msgid "Upgrading"
msgstr "अपग्रेड करतोय"

#: ../DistUpgrade/DistUpgradeController.py:2006
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2007
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2028
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2029
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2051
#: ../DistUpgrade/DistUpgradeController.py:2102
#: ../DistUpgrade/DistUpgradeController.py:2108
#: ../DistUpgrade/DistUpgradeController.py:2119
msgid "Upgrade complete"
msgstr "अपग्रेड पूर्ण"

#: ../DistUpgrade/DistUpgradeController.py:2052
#: ../DistUpgrade/DistUpgradeController.py:2103
#: ../DistUpgrade/DistUpgradeController.py:2109
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "अपग्रेड पूर्ण झालाय पण अपग्रेड क्रियेमधे त्रुटी आल्या होत्या."

#: ../DistUpgrade/DistUpgradeController.py:2071
msgid "System upgrade is complete."
msgstr "सिस्टिम अपग्रेड संपूर्ण"

#: ../DistUpgrade/DistUpgradeController.py:2120
msgid "The partial upgrade was completed."
msgstr "आंशिक अपग्रेड पूर्ण झालाय"

#: ../DistUpgrade/DistUpgradeController.py:2148
msgid "Lock screen disabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2149
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "प्रकाशन टिप शोधू शकत नाही"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "सर्व्हर ओव्हरलोड झाला असेल. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "प्रकाशन टिप डाउनलोड करू शकला नाही"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "कृपया तुमचे इंटरनेट कनेक्टशन तपासा."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "'%(file)s' चे प्रमाणीकृत विरुद्ध '%(signature)s' करा "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "'%s' बाहेर काढत आहे"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "अद्यतन साधन चालू शकले नाही"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"बहुधा सुधारणा साधन मध्ये बग आहे. आज्ञा 'ubuntu-bug ubuntu-release-upgrader-core' "
"वापरून बग म्हणून तक्रार करा."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "अद्यतन साधन सही"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "अद्यतन साधन"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "मिळविता आले नाही"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "अद्ययावत करता आले नाही. कदाचीत  network मध्ये त्रूटी असेल. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "ओळख अपयशी"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr "सुधारणा प्रमाणीकरण अयशस्वी. नेटवर्क किंवा सर्व्हरसह एक समस्या असू शकते. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "बाहेर काढण्यास अयशस्वी"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "सुधारणा अवतरण अयशस्वी. नेटवर्क किंवा सर्व्हरसह एक समस्या असू शकते. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "पडताळणी अयशस्वी झाली आहे"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"अद्ययावत करण्याची पडताळणी अयशस्वी झाली आहे.कदाचीत नेटवर्क किंवा सर्वर मध्ये त्रूटी आहे. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "अपग्रेड सुरु करू शकत नाही."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"हे सामान्यतः सिस्टीम मूळे होते जिथे  /tmp noexec म्हणून mount केले जाते.कृपया nonexec "
"सोडून पुन्हा mount करा आणि अद्ययावत करणे पुन्हा सुरु करा."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "'%s' त्रुटी संदेश आहे"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "अद्यतन करा (_U)"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "रद्द करा (&C)"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "रिलीज़ नोट्स"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "अपग्रेड"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "लागणारे अधिक पाकेजस डाऊनलोड करत आहे ..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "%2s पैकी %1s फाईल %3s B/s या गतीने"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "%2s पैकी %1s फाईल"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "कृपया '%s' ड्राइव्ह '%s' मध्ये समाविष्ट करा"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "मीडिया बदल"

#: ../DistUpgrade/DistUpgradeQuirks.py:219
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr "आपल्या ग्राफिक्स हार्डवेअर उबंटू 14.04 मध्ये पूर्णपणे समर्थीत नाही."

#: ../DistUpgrade/DistUpgradeQuirks.py:221
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:245
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "तुमचे ग्राफिक हार्डवेयर कदाचित उबुंटू १२.०४ LTS पूर्णपणे वापरू शकत नाही"

#: ../DistUpgrade/DistUpgradeQuirks.py:247
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:267
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:271
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"हा संगणक सध्या AMD 'fglrx' ग्राफिक driver वापरत आहे.उबुंटू १०.०४ LTS मध्ये तुमचे "
"हार्डवेयर वापरण्या साठी ह्या driver ची कोणतीही आवृत्ती उपलभ्ध नाही.\n"
"तुम्हाला पुढे जायचे?"

#: ../DistUpgrade/DistUpgradeQuirks.py:310
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:311
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:326
msgid "No ARMv6 CPU"
msgstr "हा ARMv6 CPU नाही"

#: ../DistUpgrade/DistUpgradeQuirks.py:327
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"तुमच्या संगणकाचा ARM CPU हा ARMv6 ह्या आवृत्ती पेक्षा जुना आहे.सगळ्या प्रणाली karmic मध्ये "
"कमीतकमी ARMv6 ला गृहीत धरून बनवल्या आहेत.ह्या हार्डवेयर वर तुम्हाला उबुंटू अद्ययावत करता "
"येणार नाही."

#: ../DistUpgrade/DistUpgradeQuirks.py:347
msgid "No init available"
msgstr "init उपलब्ध नाही"

#: ../DistUpgrade/DistUpgradeQuirks.py:348
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:461
msgid "Connection to the Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:462
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:477
msgid "Connection to Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:478
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:487
msgid "Outdated snapd package"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:488
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:511
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:542
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:548
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:552
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:555
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:584
msgid "PAE not enabled"
msgstr "PAE अक्षम आहे"

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:895
msgid "Checking for installed snaps"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "अद्यतन संकुले शोधण्यासाठी देण्यात आलेला सीडी-रॉम मार्ग वापरावा"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"फ्रन्टएन्ड वापरा. सध्या उपलब्ध: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*नाकारले* हा पर्याय दुर्लक्षित केला जाईल"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "GNU स्क्रीन समर्थन अक्षम करा"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "datadir सेट करा"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "विकास प्रकाशनवर श्रेणीसुधारित करा"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr ""

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "बदल करत आहे"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "अवलंबून असलेल्या समस्यांमुळे - ज़ुळवणी न करिता सोडून देते"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "'%s' स्थापित होऊ शकले नाही"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "'diff ' आज्ञा सापडली नाही"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "काहीतरी जबर घोळ झाला"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl-c दाबले"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "(%s) स्थापा"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "(%s) अद्ययावत"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "फरक दाखवा. >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< फरक लपवा"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "त्रुटी"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "बंद करा (&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "टर्मिनल दाखवा >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< टर्मिनल लपवा"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "माहिती"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "तपशील"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "%s काढून टाका"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "काढा(आपोआप प्रस्थापित केले होते) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "इन्स्टॉल %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "अद्ययावत %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "आवश्यक असलेले पुन्हा सुरु करा"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "प्रणाली बंद करून परत सुरु करा."

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>सुरु असलेले अद्ययावत रद्द करा?</big></b>\n"
"अद्ययावत करणे रद्द केल्यास सिस्टीम अस्थिर होऊ शकते.अद्ययावत करणे सुरु ठेवा हा तुम्हाला ठाम "
"सल्ला आहे ."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "अपग्रेड थांबवायचा का?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li दिवस"
msgstr[1] "%li दिवस"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li तास"
msgstr[1] "%liतास"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li मिनिट"
msgstr[1] "%li मिनिटे"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li सेकंद"
msgstr[1] "%li सेकंद"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr ""

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr ""

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr ""

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "अद्ययावत करण्यासाठी तयार होत आहे"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "नवीन पाकेजस मिळवत आहे"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "साफ करत आहे"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
msgstr[1] ""

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d पाकेज काढून टाकले जाणार आहे"
msgstr[1] "%d पाकेजस काढून टाकले जाणार आहे"

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d नवीन पाकेज प्रस्थापित केले जाणार आहे."
msgstr[1] "%d नवीन पाकेजस प्रस्थापित केले जाणार आहे."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d पाकेज अद्ययावत केले जाणार आहे."
msgstr[1] "%d पाकेजस अद्ययावत केले जाणार आहे."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"तुम्हाला एकून %s डाऊनलोड करावे लागेल. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"अद्ययावत करण्यास कित्येक तास लागू शकतात.एकदा डाऊनलोड झाल्यावर क्रिया थांबविता येणार "
"नाही."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "पाकेजस काढून टाकण्यास कित्येक तास लागू शकतात. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "तुमच्या प्रणालीसाठी अपग्रेडस उपलब्ध नाहीत. हा अपग्रेड थांबवला जाईल."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "पुन्हा सुरु करणे आवश्यक आहे"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "अद्ययावत करणे संपले आहे,पुन्हा सुरु करणे आवश्यक आहे.तुम्हाला हे आता करायचे आहे ?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "अयशस्वी झाल्यामुळे बंद करीत आहे"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "सुरु ठेवण्यासाठी कृपया [ENTER] दाबा"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "सुरु ठेवा(हो/नाही) "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "तपशील[d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "हो"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "नाही"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "काढून टाका: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "%s प्रस्थापित करा.\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "अपग्रेड: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "पुढे चला[Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"अद्ययावत करणे पूर्ण करण्यासाठी पुन्हा सुरु करणे आवश्यक आहे. तुम्ही ' हो ' निवडल्यास सिस्टीम "
"पुन्हा सुरु होईल."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "फाईल डाऊनलोड %(total)li पैकी %(current)li आणि %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "फाईल डाऊनलोड %(total)li पैकी %(current)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "प्रत्येक फाईलची प्रगती दाखवा."

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_अद्ययावत करणे रद्द करा"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_अद्ययावत करणे चालू ठेवा"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr "<b><big>सुरु असलेले अद्ययावत करणे रद्द करायचे का?  </big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "अपग्रेड सुरु करा"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "अदलाबदल करा (_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "फाईल्समधील फरक"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "त्रूटी कळवा (_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "पुढे चला"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "अपग्रेड सुरु करायचा का?"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>अपग्रेड पूर्ण करण्यासाठी प्रणाली बंद करून सुरु करा.</big></b>\n"
"\n"
"पुढे जाण्याआधी तुमचे काम सेव्ह करा."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "संगणक पुन्हा सुरु करत आहे"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "टर्मिनल"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "अपग्रेड"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>उबुंटूचे नवीन वर्जन उपलब्ध आहे. अपग्रेड करायचे का?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "अपग्रेड करू नका."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "नंतर विचारा."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "हो, आता अपग्रेड करा."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "तुम्ही नवीन उबुंटूला अपग्रेड करायला नकार दिला आहे."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr ""

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "फक्त वर्जन दाखवा."

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr ""

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr ""

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr ""

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr ""

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"वितरणाची नवी रिलिज उपलब्ध आहे का एव्हढेच फक्त तपास आणि exit code द्वारे उत्तर कळव"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr ""

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr ""

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "उबुंटूची नवीन आवृत्ती उपलब्ध आहे का ह्याचा शोध चालू आहे"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr ""

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr ""

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr ""

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr ""

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr ""

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "नवीन रिलीज %s उपलब्ध आहे."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "त्याला अपग्रेड करण्यासाठी 'do-release-upgrade' चालवा."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr ""

#: ../check_new_release_gtk.py:113
#, fuzzy, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "उबुंटू %s(वर्जन) अपग्रेड उपलब्ध"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "तुम्ही उबुंटूला अपग्रेड करण्यास नकार करत आहात. %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr ""

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "अपग्रेड करीत असताना एक अडचण येत आहे:\n"
#~ "%s\n"
#~ "\n"
#~ "ही अडचण बहुदा खालील कारणांमुळे येत असावी:\n"
#~ " * तुम्ही उबुन्टूच्या प्री-रीलिज आवृत्तीसाठी अपग्रेड करत असाल\n"
#~ " * उबुन्टूची सध्याची प्री-रीलिज आवृत्ती तुम्ही तुमच्या संगणकावर चालवत असाल\n"
#~ " * उबुन्टूकडून न पुरविल्या गेलेल्या अनधिकृत सॉफ्टवेअर पॅकेजेस मुळे\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "तुमच्या सिस्टीम वर ubuntu-desktop, kubuntu-desktop, xubuntu-desktop किंवा "
#~ "edubuntu-desktop यांपैकी कोणतेच पॅकेज नाही आणि तुम्ही सध्या उबुन्टूची कोणती आवृत्ती "
#~ "तुमच्या संगणकावर सध्या चालवत आहात याबद्दल माहिती मिळवणे अशक्य होते.\n"
#~ " कृपया पुढे जाण्याअगोदर वरीलपैकी कोणतेही एखादे पॅकेज synaptic किंवा apt-get द्वारे "
#~ "स्थापित करा."

#~ msgid "Sandbox setup failed"
#~ msgstr "Sandbox चे सेटअप अयशस्वी झाले"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "sandbox वातावरणनिर्मिती तयार करणे शक्य नव्हते."

#~ msgid "Sandbox mode"
#~ msgstr "Sandbox मोड"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "तुमची python ची स्थापना बिघडली आहे किंवा दूषित झाली आहे. कृपया '/usr/bin/python' "
#~ "symlink दुरुस्त करा."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "'debsig-verify' पॅकेज स्थापित केले आहे"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "त्या स्थापित केलेल्या पॅकेजसह अपग्रेड पुढे चालू शकत नाही.\n"
#~ "प्रथमतः ते पॅकेज synaptic द्वारे किंवा 'apt-get remove debsig-verify' द्वारे काढून "
#~ "टाका आणि पुन्हा एकदा अपग्रेड चालू करा."

#~ msgid "No new release found"
#~ msgstr "एकही नवीन रिलीज मिळाली नाही."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "अपग्रेडबाहेर पडलेलो आहे.अपग्रेडसाठी '%s' डिस्कवर एकुण '%s' रिकामी जागा पाहिजे. '%s' "
#~ "वर अजुन कमीतकमी '%s' एवढी जागा रिकामी करा. 'केराची टोपली' रिकामी करा आणि "
#~ "'sudo apt-get clean' करुन जुन्या इनस्टॉलेशनच्या वेळचे कामचलाऊ पॅकेजेस काढुन टाका."

#~ msgid "No i686 CPU"
#~ msgstr "हा i686 CPU नाही"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "हा अपग्रेड sandbox (चाचणी) मोडमधे सुरु आहे. सर्व बदल '%s' वर लिहिले जातात आणि "
#~ "पुढल्या रीबूटनंतर ते नाहिसे होतील.\n"
#~ "\n"
#~ "आतापासुन ते पुढल्या रीबूटपर्यंत सिस्टिम डिरेक्टरीवर लिहिले गेलेले *कोणतेही* बदल "
#~ "कायमस्वरुपाचे नाहीत."

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "जर यातले काहीही लागू होत नसल्यास, नंतर एक टर्मिनल मध्ये आदेश 'ubuntu-bug ubuntu-"
#~ "release-upgrader-core' वापरून या बगची तक्रार करा."

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "सुधारणा गणना करताना न सोडविण्याजोगी समस्या आली.\n"
#~ "\n"
#~ "  हे यामुळे देखील होऊ शकते:\n"
#~ "  * उबंटूची एक पूर्व प्रकाशन आवृत्ती श्रेणीसुधारित करणे\n"
#~ "  * उबंटूची वर्तमान पूर्व प्रकाशन आवृत्ती चालवणे\n"
#~ "  * उबंटू द्वारे प्रदान न केल्या गेलेले अनधिकृत सॉफ्टवेअर संकुल चालवणे\n"
#~ "\n"

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "aufs वापरून सॅन्डबॉक्स अद्यतन"
